README.md

```diff
@@ -16,3 +16,25 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+-----
+
+To verify the claim, I could create an algorithm that generates a random list of two elements, a random list of three elements, etc. up to a random list of 1000 elements. The algorithm would give X and a general sorting algorithm the same of each list and keep track of how long it takes the sorting algorithms to sort each list. It would do this by starting two timers, one for each algorithm, when it gives both sorting algorithms the 2 element list and would stop an algorithm’s timer when it sorted all of the lists. Once both algorithms sorted all lists, the times would be compared. If the general sorting algorithm is slower by at least log(n), that means the claim is true since X would be asymptotically faster.
+
+To tell if the general sorting algorithm is slower by at least log(n), I could create a graph, where the x-axis represents the input size and the y-axis represents the time it takes for the algorithm to sort the list.  I could then compare the lines of the general sorting algorithm with X, and if the line of X follows a linear trend and the curve produced from the general sorting algorithm's runtime increases at a higher rate than X, eventually overtaking the X curve, then that means the general sorting algorithm is slower by at least log n.
```

Comment:
The key thing here seems to be the linear trend -- do you need to compare it to another sorting algorithm at all?